examples/timestamp/signed_timestamp.c

```diff
@@ -282,6 +294,10 @@ int TPM2_Timestamp_Test(void* userCtx)
     }
 
     /* Close key handles */
+    if (!tryNVkey) {
+        wolfTPM2_UnloadHandle(&dev, &storage.handle);
```

Comment:
I would expect you'd still need to unload the handle in either case. If you call  `wolfTPM2_CreateSRK` it will return a valid transitent (temp) handle in `storage.handle.hndl`.

Reply:
It wasn't being unloaded before unless `wolfTPM2_NVStoreKey()` failed. Does that call essentially invalidate the handle?
See above: https://github.com/wolfSSL/wolfTPM/pull/127/files/62ff638dc7937d94f19e2ebf96e4b769fcf59113#diff-be84e1391f63202a3163ecb899bcddc655a28ce91b092f20ab55971aac712190L131-L136

Reply:
I read that wrong, all good here. You are unloading it in the Win TBS case. Awesome